projects/demo/src/modules/components/slider-old/examples/import/import-module.txt

```diff
@@ -0,0 +1,13 @@
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
```

Comment:
what do you think about using markdown? as it is in the example with the avatar demo